RoboJackets/robocup-software
# Documentation for ER-Force Simulator Installation

## Description
Created documentation on how to install the ER-Force simulator. It should show up on the HTML version of our documentation.

## Steps to test
### Test Case 1
1. source ROS setup file and local setup
2. type into terminal: make open-docs
3. check the information in the documentation is correct and makes sense

Expected result: should work and make sense
